### ISSUE TITLE
Deployment failed - Automation

### ISSUE THREAD

**Issue by isotonicuk** [stgeorgi/wvdquickstart#18]
Hi
I have tried to create an ARM deployment using Azure Devops setup via www.wvdquickstart.com. I have an AD server already created with domain admin privileges with a domain join account. This is in a separate resource group with a separate vnet.

The deployment I am creating is in a different resource group but in the same re
[Deployment-Microsoft.Template-20201104193615.zip](https://github.com/stgeorgi/wvdquickstart/files/5490326/Deployment-Microsoft.Template-20201104193615.zip)
gion.

I attach the deployment error zip file.

**Comment by stgeorgi** [comment]
It looks like there is no piering between the two VNETs (the one with dc and the one where new host pools are being deployed), is my observation correct?

**Comment by isotonicuk** [comment]
> It looks like there is no piering between the two VNETs (the one with dc and the one where new host pools are being deployed), is my observation correct?

Ok thanks, how would I check and enable peering?

**Comment by justin-shiah** [comment]
@isotonicuk If you go to your VNet, under settings there should be a Peerings option. If you click on it, you can see all existing peerings or create a new peering.

**Comment by isotonicuk** [comment]
Ok great, so effectively this just joins the vnets together as they are currently in different resource groups?

**Comment by justin-shiah** [comment]
@isotonicuk You're correct in the sense that this joins the VNets together, but the issue isn't that they're in different resource groups. The point of peering the VNets is so that the second VNet can still talk to the domain controller.
